src/rust/engine/src/nodes/execute_process.rs

```diff
@@ -161,7 +161,7 @@ impl ExecuteProcess {
         let attempt = externs::getattr(value, "attempt").unwrap_or(0);
 
         Ok(Process {
-            argv: externs::getattr(value, "argv").unwrap(),
+            argv: externs::getattr(value, "argv")?,
```

Comment:
No need to unwrap since we can just return an error. (And panics in the engine end up returning less useful error backtraces in any event.)